Testscripts/Linux/STOR-Large-CopyFile.sh

```diff
@@ -0,0 +1,223 @@
+        mkfs.$fs  ${driveName}1
+        if [ "$?" = "0" ]; then
+            LogMsg "mkfs.${fs}   ${driveName}1 successful..."
+            mount ${driveName}1 /mnt
+            if [ "$?" = "0" ]; then
+                LogMsg "Drive mounted successfully..."
+
+                # step 1: test for local copy file
+                if [[ $TestLocalCopy = "True" ]]; then
+                     LogMsg "Start to test local copy file"
+                     Test_Local_Copy_File
+                fi
+
+                if [[ $TestWget = "True" ]]; then
+                # step 2: wget 5GB file to disk
+                     LogMsg "Start to test wget file"
+                     Test_Wget_File
+                fi
+
+                # step 3: mount nfs file, then copy file to disk
+                if [[ $TestNFSCopy = "True" ]]; then
+                      LogMsg "Start to test copy file from nfs mout point"
+                      Test_NFS_Copy_File
+                fi
+
+                df -h
+                # umount /mnt files
+                umount /mnt
+                if [ "$?" = "0" ]; then
+                      LogMsg "Drive unmounted successfully..."
+                fi
+            else
+                LogErr "Error in mounting drive..."
+                SetTestStateFailed
+            fi
+        else
+            LogErr "Error in creating file system ${fs}.."
+            SetTestStateFailed
+        fi
+    else
+        LogErr "Error in executing parted  ${driveName}1 for ${fs}"
+        SetTestStateFailed
+    fi
+}
+
+# Source the constants file
+if [ -e ~/${CONSTANTS_FILE} ]; then
+    source ~/${CONSTANTS_FILE}
+else
+    LogErr "Error: in ${CONSTANTS_FILE} file"
+    SetTestStateFailed
+    exit 0
+fi
+
+# Create the state.txt file so ICA knows we are running
+SetTestStateRunning
+
+# Make sure the constants.sh file exists
+if [ ! -e ./constants.sh ];
+then
+    LogErr "Cannot find constants.sh file."
+    SetTestStateAborted
+    exit 0
+fi
+
+# Check for call trace log
+./check_traces.sh &
+
+# Count the number of SCSI= and IDE= entries in constants
+diskCount=0
+for entry in $(cat ./constants.sh)
+do
+    # Convert to lower case
+    lowStr="$(tr '[A-Z]' '[a-z' <<<"$entry")"
+
+    # does it start wtih ide or scsi
+    if [[ $lowStr == ide* ]];
+    then
+        diskCount=$((diskCount+1))
+    fi
+
+    if [[ $lowStr == scsi* ]];
+    then
+        diskCount=$((diskCount+1))
+    fi
+done
+
+echo "constants disk count= $diskCount"
+
+# Compute the number of sd* drives on the system
+for driveName in /dev/sd*[^0-9];
+do
+    # Skip /dev/sda
+    if [ ${driveName} = "/dev/sda" ] || [ ${driveName} = "/dev/sdb" ]; then
+        continue
+    fi
+
+    for fs in "${fileSystems[@]}"; do
+        LogMsg "Start testing filesystem: $fs"
+        StartTst=$(date +%s.%N)
+        command -v mkfs.$fs
+        if [ $? -ne 0 ]; then
+            UpdateSummary "File-system tools for $fs not present. Skipping filesystem $fs."
```

Comment:
LogMsg instead of UpdateSummary.
LE: You have almost 20 UpdateSummary occurences - please replace all of them

Reply:
ditto